### ISSUE TITLE
[Enhancement]

### ISSUE THREAD

**Issue by Unfound999** [ciderapp/Cider#109]
**Is your feature request related to a problem? Please describe.**
Whenever I use the app, all of the album art is really low resolution. I don't know if it's because it's having a problem pulling it from iTunes or what, but it just seems to be really low resolution. Also, this might be a super nit-pick, but I have a 4k monitor, and the app seems to be scaled correctly, except that it looks like it's running some sort of post-processing thing like fxaa or tsaa because it looks somewhat blurry.

**Describe the solution you'd like**
Personally, my main thing is I would like the album art to be the full resolution that it is in iTunes. It would be nice if the text could look sharper, but I don't even know if that's possible or a sensible ask.

**Describe alternatives you've considered**
Just a feature to make the album art the full resolution and, if possible, to make the text look less fuzzy. (Also, I am aware you can go into the settings and do all the dpi stuff, but that has never really worked for me when I've tried to scale a software correctly.)

**Additional context**
I have a 4k monitor, so maybe it's just that the app hasn't been correctly scaled for 4k yet, but it isn't displaying in 1080p, and it seems to be displaying in 4k, so I don't know.

And in the pictures, you can see how iTunes is a much higher resolution for the album art, and even the text and other UI elements seem sharper.

![Screenshot (2)](https://user-images.githubusercontent.com/82480013/148878617-fe53addc-1708-4750-a199-5d56165118c0.png)
![Screenshot (3)](https://user-images.githubusercontent.com/82480013/148878645-2fd55f56-2e6e-4be7-86a7-d23d814efd11.png)
![Screenshot (4)](https://user-images.githubusercontent.com/82480013/148878653-8fa640a5-29c4-4a65-87c2-913baae09111.png)
![Screenshot (5)](https://user-images.githubusercontent.com/82480013/148878665-d29e2065-4bee-4ed0-9879-9e4421b6ada9.png)

**Comment by vapormusic** [comment]
yes, it's on our end. we limit the art resolution to 300 pixels to not spike the RAM

**Comment by Unfound999** [comment]
Will it ever get to a point where you guys will be able to allow the resolution to be larger? Or will it have to stay this way?

**Comment by cryptofyre** [comment]
It's planned, as are a ton of other things. Hence the app being in Alpha.
